python/tests/stk_structures.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+
+import stk
+
+import chemiscope
+
+BASE_FRAME = stk.BuildingBlock("N#CC")
+
+
+class TestStructures(unittest.TestCase):
+    """Conversion of structure data to chemiscope JSON"""
+
+    def test_structures(self):
+        data = chemiscope.create_input(BASE_FRAME)
+        self.assertEqual(len(data["structures"]), 1)
+        self.assertEqual(data["structures"][0]["size"], 6)
+        self.assertEqual(
+            data["structures"][0]["names"],
+            ["N", "C", "C", "H", "H", "H"],
+        )
+        self.assertEqual(
+            data["structures"][0]["x"],
+            [
+                1.6991195138834223,
+                0.7737143493209756,
+                -0.41192204250544034,
+                -0.7778845126633998,
+                -1.1777543806588109,
+                -0.10527292738297804,
+            ],
```

Comment:
@andrewtarzia when running this test locally, I get an error:


```
AssertionError: Lists differ: [1.8856062905183792, 0.8568739943293787, -0.[74 chars]6763] != [1.6991195138834223, 0.7737143493209756, -0.[77 chars]7804]

First differing element 0:
1.8856062905183792
1.6991195138834223

- [1.8856062905183792,
-  0.8568739943293787,
-  -0.4418130132537843,
-  -1.224037530127693,
-  -0.6838309641680245,
-  -0.3927987773106763]
+ [1.6991195138834223,
+  0.7737143493209756,
+  -0.41192204250544034,
+  -0.7778845126633998,
+  -1.1777543806588109,
+  -0.10527292738297804]
```

Do you think this is because the orientation of the molecule is not fixed? Or that rdkit gives different results here for some reason? 

I'm running on an arm64 CPU / macOS if that's relevant

Reply:
Mm that is strange. Molecule construction in stk should be constant by setting rdkit seeds and such. That said, I don't think we've ever tested on a mac, I will check with another Mac user. Sorry!

Reply:
Yes, so Lukas confirmed that the STK tests fail on a Mac. I assume that the coordinates are self consistent, but the random numbers used differ across hardware. What's the best course of action? I could provide coordinates to the molecules instead of using rdkit conformer generation? Or can we mute a test on certain hardware?

Reply:
For the tests in chemiscope, I'll just remove the tests about the exact coordinate values, it does not matter too much to test the functionality here!